src/theme/components/created-by.module.css

```
@@ -5,6 +5,10 @@
 }
 
 .bufLogo {
+  display: flex;
+  margin-bottom: 3px;
```

I played around with this in a browser and 1px seems to align the baseline text in the logo with the text outside the logo
```suggestion
  margin-bottom: 1px;
```

---

It still kinda looks slightly off to me, but 🤷‍♂️